### ISSUE TITLE
Write unit tests for QueryPatternUtils

### ISSUE THREAD

**Issue by hartig** [LiUSemWeb/HeFQUIN#289]
There are plenty of functions in `QueryPatternUtils` for which it would be good to have a few unit tests.

**Comment by keski** [comment]
`QueryPatternUtils` has been refactored as of #428, with a lot of functionality moved into `SPARQLGraphPattern`.

However, we should still provide tests for the remaining `convertToJenaOp` and `convertToJenaElement`.

**Comment by hartig** [comment]
Makes sense! But we should also not overdo it. The number of combinations of patterns that may be considered is endless. Picking a few for the tests is fine--mainly to make sure that these two functions work in general.

**Comment by keski** [comment]
I created a single conversion test for each of the following:
- `TriplePattern`
- `BGP`
- `SPARQLUnionPattern`
- `SPARQLGroupPattern`
- `GenericSPARQLGraphPatternImpl1` (`SPARQLGraphPattern` created from an `ElementGroup`)
- `GenericSPARQLGraphPatternImpl2` (`SPARQLGraphPattern` created from an `OpExtend`)

PR created (https://github.com/LiUSemWeb/HeFQUIN/pull/435)
